tests/routes/view.test.js

```diff
@@ -513,7 +513,7 @@ describe('Routes', () => {
       '2e0afeb7-95d2-42f4-80e6-ccf96b282730'
     )
     expect(caseService.getMatchDetails).toHaveBeenCalledWith(
-      '2e0afeb7-95d2-42f4-80e6-ccf96b282730'
+      '2e0afeb7-95d2-42f4-80e6-ccf96b282730', undefined
```

Comment:
what is this `undefined`? is it something to do with the showAllMatches? If so could it be changed to a Boolean?

Reply:
it is a second param that when not passed, defaults to undefined. This is in-place of showAllMatches.

Reply:
Is this the call which is being mocked here?
`const response = await getMatchDetails(defendantId, queryParams.showAllMatches)`

If so, it is followed a few lines later in the same method by 
`const showAllMatches = Boolean(queryParams.showAllMatches)`

How about moving `const showAllMatches` up a bit and passing it into `getMatchDetails` so that it's always either `true` or `false`?
That way you don't need to do `showAllMatches = Boolean(showAllMatches)` at the top of `getMatchDetails`

As far as I can see it is only called from server/routes/handlers/getMatchingRecordRouteHandler.js

Reply:
have made the changes as per the above suggestion